packages/dm-core-plugins/blueprints/list/FunctionalityConfig.json

```diff
@@ -34,6 +34,20 @@
       "attributeType": "boolean",
       "optional": true,
       "default": true
+    },
+    {
+      "name": "open",
+      "type": "CORE:BlueprintAttribute",
+      "attributeType": "boolean",
+      "optional": true,
+      "default": false
+    },
+    {
+      "name": "expand",
+      "type": "CORE:BlueprintAttribute",
+      "attributeType": "boolean",
+      "optional": true,
+      "default": true
```

Comment:
We can adjust this, maybe default false is better default values, but to avoid updating all tests keep the default values as before. So expand is default true, and open is default false.